fuel-client/src/client.rs

```diff
@@ -534,6 +537,24 @@ impl FuelClient {
 
         Ok(messages)
     }
+
+    /// Request a merkle proof of an output message.
+    pub async fn output_proof(
```

Comment:
Maybe we should call this `message_output_proof`? As we may have other kinds of proofs in the future.